src/index.ts

```diff
@@ -13,5 +19,5 @@ export const connect = (url) => new Promise((fulfill, reject) => {
   })
 })
 
-const win = (window as any)
-win.holoclient = win.holoclient || {connect}
+// const win = (window as any)
+// win.holoclient = win.holoclient || {connect}
```

Comment:
Didn't we say we were going to have this as a proper module as well as a browser-only thing that goes into the global window scope?

Reply:
Is manipulating the window directly inside the module the only way to make functions available to use in the browser? It seems like a hack but knowing web dev it might be the best way of doing it. 

If this is the best way to do it then we should definitely add logic to include that in the browser-only build.